app/scripts/controllers/main.js

```diff
@@ -7,11 +7,75 @@
  * # MainCtrl
  * Controller of the todoApp
  */
+//Initializing AngularJS app by creating a module, app name is 'todoApp'
+
+ //Creating a controller named 'MainCtrl' and adding it to our app
+ //adding '$scope' into the function so that our view has access to the controller's
+ //data. This is called dependency injection
+
 angular.module('todoApp')
-  .controller('MainCtrl', function () {
-    this.awesomeThings = [
-      'HTML5 Boilerplate',
-      'AngularJS',
-      'Karma'
-    ];
-  });
+  .controller('MainCtrl', function ($scope, $window, Task, ngNotify) {
+
+    //Getting user token and adding it to scope
+    $scope.userToken = localStorage.getItem("token");
+
+    //Redirect user to Authentication page
+    $scope.toAuthpage = function(){
+      $window.location.href = '#/auth';
+    }
+
+
+    //Initializing variable to false for showing archive notes
+    $scope.showArchive = false;
+
+    //making an object of todos notes in the object of todos list
+    $scope.todoList = Task.get({"token":$scope.userToken}, function(response){
+      console.log("got tasks");
+    }, function(err){
+      console.log("did not obtain tasks");
+    });
+    //Initializing a blank note for ng-model in jumbotron of index.html
+    $scope.note = {
+      title: "",
+      body: "",
+      date: ""
+    };
+
+    //add note to todoList
+    $scope.addNote = function(noteTitle, noteBody){
+      var serverNode = Task.create({"title": noteTitle,"body":noteBody ,"token":$scope.userToken}, function(response){
```

Comment:
Firstly, serverNode is a typo and should be serverNote.

Secondly, separate out this inline JSON like I described above.